### DISCUSSION TITLE
Help w/ custom firmware E3V2

### DISCUSSION THREAD

**Discussion by Magutanko** [Jyers/Marlin#1653] [Q&A]
Hey,
I'm wondering if anyone can help compile a firmware for me, as I'm not so confident on this side of things. 

I have an E3V2, 4.2.2 board, but with an "Ender Extender" kit. Making my print area 400x400x450.

I was hoping for this latest release, but with manual mesh level enabled also.

Is that possible at all? 
I will also need one for BLTouch eventually as well. 

Thank you to anyone that can help, I'd be happy to buy you a coffee via PayPal! 😊

**Comment by Cal-LF** [comment]
Try this.
[E3V2 4.2.2 400x400x450 Manual Mesh 5x5.zip](https://github.com/Jyers/Marlin/files/7613695/E3V2.4.2.2.400x400x450.Manual.Mesh.5x5.zip)

You can also join Jyers patreon to get custom firmware made automatically. https://www.patreon.com/jyers
I encourage you to try compiling a firmware. After installing VS Code and the two extensions PlatformIO and Autobuild Marlin, it is a matter of editing text files Configuration.h and Configuration_adv.h then compiling. If you need assistance I will guide you.

**Comment by Magutanko** [comment]
Thank You kind stranger! 
This worked perfectly, and though I have a little experience with compiling already - I managed to make a basic marlin firmware with the new bed sizes, I couldnt figure out how to get manual mesh working. 
I've ordered a BLTouch so when that arrives I may need to add that as well, I will try my hand at it. 

I'm new to a lot of this, but im learning in leaps and bounds!
I will consider the patreon, as my machine is a bit modified - it could be useful for future updates! 

Once again, thanks
